src/main.py

```diff
@@ -1,3 +1,27 @@
+# This file is a part of nylo
+#
+# Copyright (c) 2018 The nylo Authors (see AUTHORS)
```

Comment:
Just found out that there's no `AUTHORS` file, I'm going to fix.

Reply:
Please, I'd prefer not to have AUTHORS and CONTRIBUTORS files.

Reply:
Please, I'd prefer not to have AUTHORS and CONTRIBUTORS files.